src/main/java/duke/task/recipetasks/RecipeTitle.java

```diff
@@ -0,0 +1,28 @@
+package duke.task.recipetasks;
+
+public class RecipeTitle {
+
+    private String recipeTitle;
+    private int index;
```

Comment:
Why do you need a separate class for title of the receipt? and what is the information stored in the index? Can an index differ for the same title string